### PR TITLE
add v5.4.3 release notes

### PR DESCRIPTION
**Pull request by Oreoxmt**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-5.4.3.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        - (dup) 修复 TiDB Lightning 不支持 Parquet 文件中以斜线 (`/`)、数字、非 ASCII 字符开头的特殊列名的问题 [#36980](https://github.com/pingcap/tidb/issues/36980)
+        - (dup) 修复了使用 IPv6 host 时无法连接到 TiDB 的问题 [#35880](https://github.com/pingcap/tidb/issues/35880)
+
+    + TiDB Data Migration (DM) **owner: @niubell**
+
+        - (dup) 修复 DM 报错 `Specified key was too long` 的问题 [#5315](https://github.com/pingcap/tiflow/issues/5315)
+        - (dup) 修复数据同步过程中，latin1 字符集数据可能损坏的问题 [#7028](https://github.com/pingcap/tiflow/issues/7028)
+        - (dup) 修复 DM IPv6 支持问题 [#6249](https://github.com/pingcap/tiflow/issues/6249)
+        - (dup) 修复 `query-status` 内可能存在 data race 的问题 [#4811](https://github.com/pingcap/tiflow/issues/4811)
+        - (dup) 修复 relay 报错时可能导致 goroutine 泄露问题 [#6193](https://github.com/pingcap/tiflow/issues/6193)
```
Comment (niubell) [inline releases/release-5.4.3.md:125]: 增加一条:
修复 DM Worker 获取 DB 连接时可能卡住的问题 [#3733](https://github.com/pingcap/tiflow/issues/3733)

File: releases/release-5.4.3.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+    - (dup) 修复使用包含 `NULL` 值的列创建主键时导致崩溃的问题 [#5859](https://github.com/pingcap/tiflash/issues/5859)
+
++ Tools
+
+    + TiDB Lightning **owner: @niubell**
+
+        - (dup) 修复 TiDB Lightning 不支持 Parquet 文件中以斜线 (`/`)、数字、非 ASCII 字符开头的特殊列名的问题 [#36980](https://github.com/pingcap/tidb/issues/36980)
+        - (dup) 修复了使用 IPv6 host 时无法连接到 TiDB 的问题 [#35880](https://github.com/pingcap/tidb/issues/35880)
+
```
Comment (niubell) [inline releases/release-5.4.3.md:118]: 新增：
修复 BigInt 自增列可能越界的问题 [#27397](https://github.com/pingcap/tidb/issues/27937)
修复去重过程中如果下游 Schema 发生变更可能会导致 Panic 的问题 [#34163](https://github.com/pingcap/tidb/issues/34163)

**Review by niubell — APPROVED** [review]
LGTM

File: releases/release-5.4.3.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+        - (dup) 修复 DM 报错 `Specified key was too long` 的问题 [#5315](https://github.com/pingcap/tiflow/issues/5315)
+        - (dup) 修复数据同步过程中，latin1 字符集数据可能损坏的问题 [#7028](https://github.com/pingcap/tiflow/issues/7028)
+        - (dup) 修复 DM IPv6 支持问题 [#6249](https://github.com/pingcap/tiflow/issues/6249)
+        - (dup) 修复 `query-status` 内可能存在 data race 的问题 [#4811](https://github.com/pingcap/tiflow/issues/4811)
+        - (dup) 修复 relay 报错时可能导致 goroutine 泄露问题 [#6193](https://github.com/pingcap/tiflow/issues/6193)
+
+    + TiCDC **owner: @nongfushanquan**
+
+        - (dup) 修复 `enable-old-value = false` 时可能出现的 cdc panic 问题 [#6198](https://github.com/pingcap/tiflow/issues/6198)
+
```
Comment (nongfushanquan) [inline releases/release-5.4.3.md:133]: (dup) 解决多 Region 场景同时下，grouting 数量过多问题[#5610](https://github.com/pingcap/tiflow/issues/5610)

File: releases/release-5.4.3.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
++ PD **owner: @nolouch**
+
+    - Fix a bug that could not handle dashboard proxy requests correctly [#5321](https://github.com/tikv/pd/issues/5321)
+    - (dup) 修复已清除的 `tombstone store` 信息在切换 PD leader 后再次出现的问题 [#4941](https://github.com/tikv/pd/issues/4941)
+    - (dup) 修复 PD 可能没创建 TiFlash Learner 副本的问题 [#5401](https://github.com/tikv/pd/issues/5401)
+
++ TiFlash
+
+    <!--compute **owner: @zanmato1984**-->
```
Comment (XuHuaiyu) [inline releases/release-5.4.3.md:103]: LGTM

File: releases/release-5.4.3.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
++ TiDB
+
+    <!--sql-infra **owner: @zimulala**-->
+    - (dup) 修复 `SHOW CREATE PLACEMENT POLICY` 输出结果不正确的问题 [#37526](https://github.com/pingcap/tidb/issues/37526)
+    - (dup) 修复集群的 PD 节点被替换后一些 DDL 语句会卡住一段时间的问题 [#33908](https://github.com/pingcap/tidb/issues/33908)
+    - (dup) 修复 `KILL TIDB` 在空闲连接上无法立即生效的问题 [#24031](https://github.com/pingcap/tidb/issues/24031)
+    - (dup) 修复 TiDB 在执行 `SHOW COLUMNS` 时会发出协处理器请求的问题 [#36496](https://github.com/pingcap/tidb/issues/36496)
+    - (dup) 修复了在开启 TiDB Binlog 时，TiDB 执行 `ALTER SEQUENCE` 会产生错误的元信息版本号，进而导致 Drainer 报错退出的问题 [#36276](https://github.com/pingcap/tidb/issues/36276)
+
+    <!--execution **owner: @zanmato1984**-->
```
Comment (XuHuaiyu) [inline releases/release-5.4.3.md:65]: LGTM

**Review by XuHuaiyu — APPROVED** [review]

**Review by nongfushanquan — APPROVED** [review]

File: releases/release-5.4.3.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+
+## Bug 修复
+
++ TiDB
+
+    <!--sql-infra **owner: @zimulala**-->
+    - (dup) 修复 `SHOW CREATE PLACEMENT POLICY` 输出结果不正确的问题 [#37526](https://github.com/pingcap/tidb/issues/37526)
+    - (dup) 修复集群的 PD 节点被替换后一些 DDL 语句会卡住一段时间的问题 [#33908](https://github.com/pingcap/tidb/issues/33908)
+    - (dup) 修复 `KILL TIDB` 在空闲连接上无法立即生效的问题 [#24031](https://github.com/pingcap/tidb/issues/24031)
+    - (dup) 修复 TiDB 在执行 `SHOW COLUMNS` 时会发出协处理器请求的问题 [#36496](https://github.com/pingcap/tidb/issues/36496)
```
Comment (zimulala) [inline releases/release-5.4.3.md:62]:
```suggestion
    - (dup) 修复 TiDB 执行 `SHOW COLUMNS` 的 `DATA_TYPE` 和 `COLUMN_TYPE` 列结果不正确的问题 [#36496](https://github.com/pingcap/tidb/issues/36496)
```

**Review by 3pointer — APPROVED** [review]

File: releases/release-5.4.3.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+
+    <!--storage **owner: @flowbehappy**-->
+
+    - (dup) 修复使用包含 `NULL` 值的列创建主键时导致崩溃的问题 [#5859](https://github.com/pingcap/tiflash/issues/5859)
+
++ Tools
+
+    + TiDB Lightning **owner: @niubell, TW: @ran-huang**
+
+        - 修复 BigInt 自增列可能越界的问题 [#27397](https://github.com/pingcap/tidb/issues/27937)
```
Comment (ran-huang) [inline releases/release-5.4.3.md:116]:
```suggestion
        - 修复 `BIGINT` 类型自增列可能越界的问题 [#27397](https://github.com/pingcap/tidb/issues/27937)
```

**Review by zimulala — APPROVED** [review]

File: releases/release-5.4.3.md [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+    - (dup) 修复带 DML 算子的 `EXPLAIN ANALYZE` 语句可能在事务提交完成前返回结果的问题 [#37373](https://github.com/pingcap/tidb/issues/37373)
+    - (dup) 修复开启 TiDB Binlog 后插入重复数据导致 data and columnID count not match 错误的问题 [#33608](https://github.com/pingcap/tidb/issues/33608)
+
+    <!--planner **owner: @winoros**-->
+
+    - (dup) 修复 Static Partition Prune 模式下带聚合条件的 SQL 语句在表为空时结果错误的问题 [#35295](https://github.com/pingcap/tidb/issues/35295)
+    - (dup) 修复 TiDB 在执行 `UPDATE` 语句时可能会 panic 的问题 [#32311](https://github.com/pingcap/tidb/issues/32311)
+    - (dup) 修复 `UnionScan` 无法保序导致的查询结果不正确的问题 [#33175](https://github.com/pingcap/tidb/issues/33175)
+    - (dup) 修复在某些情况下 `UPDATE` 语句错误地消除了 projection 导致 `Can't find column` 报错的问题 [#37568](https://github.com/pingcap/tidb/issues/37568)
+    - (dup) 修复某些情况下分区表无法充分利用索引来扫描数据的问题 [#33966](https://github.com/pingcap/tidb/issues/33966)
```
Comment (winoros) [inline releases/release-5.4.3.md:84]:
```suggestion
    - (dup) 修复某些情况下分区表无法充分利用索引来扫描数据的问题 [#33966](https://github.com/pingcap/tidb/issues/33966)
    - 修复特定情况下，`EXECUTE` 命令可能抛出异常的问题 [#37187](https://github.com/pingcap/tidb/issues/37187)
    - 修复开启 prepare plan cache 时，使用 `BIT` 类型的索引可能会查询结果错误的问题 [#33067](https://github.com/pingcap/tidb/issues/33067)
    
```

File: releases/release-5.4.3.md [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+    <!--execution **owner: @zanmato1984**-->
+
+    - (dup) 修复 `UNION` 运算符可能会非预期地返回空结果的问题 [#36903](https://github.com/pingcap/tidb/issues/36903)
+    - (dup) 修复在 TiFlash 中为分区表开启动态模式时结果出错的问题 [#37254](https://github.com/pingcap/tidb/issues/37254)
+    - (dup) 修复 `INL_HASH_JOIN` 和 `LIMIT` 一起使用时可能会卡住的问题 [#35638](https://github.com/pingcap/tidb/issues/35638)
+    - (dup) 修复执行 `SHOW WARNINGS` 时可能会报 `invalid memory address or nil pointer dereference` 的问题 [#31569](https://github.com/pingcap/tidb/issues/31569)
+
+    <!--transaction **owner: @cfzjywxk**-->
+
+    - Fix `invalid transaction` error when doing stale read in RC isolation level [#30872](https://github.com/pingcap/tidb/issues/30872)
```
Comment (TomShawn) [inline releases/release-5.4.3.md:74]:
```suggestion
    - 修复在读已提交隔离级别 (RC isolation level) 执行 Stale Read 报 `invalid transaction` 错误的问题 [#30872](https://github.com/pingcap/tidb/issues/30872)
```

File: releases/release-5.4.3.md [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+
+    <!--planner **owner: @winoros**-->
+
+    - (dup) 修复 Static Partition Prune 模式下带聚合条件的 SQL 语句在表为空时结果错误的问题 [#35295](https://github.com/pingcap/tidb/issues/35295)
+    - (dup) 修复 TiDB 在执行 `UPDATE` 语句时可能会 panic 的问题 [#32311](https://github.com/pingcap/tidb/issues/32311)
+    - (dup) 修复 `UnionScan` 无法保序导致的查询结果不正确的问题 [#33175](https://github.com/pingcap/tidb/issues/33175)
+    - (dup) 修复在某些情况下 `UPDATE` 语句错误地消除了 projection 导致 `Can't find column` 报错的问题 [#37568](https://github.com/pingcap/tidb/issues/37568)
+    - (dup) 修复某些情况下分区表无法充分利用索引来扫描数据的问题 [#33966](https://github.com/pingcap/tidb/issues/33966)
+    - 修复某些情况下，`EXECUTE` 命令可能抛出异常的问题 [#37187](https://github.com/pingcap/tidb/issues/37187)
+    - 修复开启 prepared plan cache 时，使用 `BIT` 类型的索引可能会导致查询结果错误的问题 [#33067](https://github.com/pingcap/tidb/issues/33067)
```
Comment (TomShawn) [inline releases/release-5.4.3.md:86]:
```suggestion
    - 修复某些情况下，`EXECUTE` 语句可能抛出非预期异常的问题 [#37187](https://github.com/pingcap/tidb/issues/37187)
    - 修复开启 Prepared Plan Cache 后，使用 `BIT` 类型的索引可能会导致查询结果错误的问题 [#33067](https://github.com/pingcap/tidb/issues/33067)
```

File: releases/release-5.4.3.md [hunk 11]
```diff
@@ -0,0 +1,145 @@
...
+
+    <!--diagnosis **owner: @mornyx**-->
+
++ TiKV **owner: @tabokie, TW: @Oreoxmt**
+
+    - 修复 Web 身份提供程序报错并失效自动恢复默认提供程序时权限拒绝的问题 [#13122](https://github.com/tikv/tikv/issues/13122)
```
Comment (TomShawn) [inline releases/release-5.4.3.md:92]:
```suggestion
    - 修复 Web 身份提供程序 (web identity provider) 报错并失效后，自动恢复为默认提供程序 (default provider) 时出现权限拒绝的问题 [#13122](https://github.com/tikv/tikv/issues/13122)
```

**Review by ran-huang — APPROVED** [review]

**Review by shichun-0415 — APPROVED** [review]

File: releases/release-5.4.3.md [hunk 12]
```diff
@@ -0,0 +1,145 @@
...
+---
+
+# TiDB 5.4.3 Release Notes
+
+发版日期：2022 年 x 月 x 日
+
+TiDB 版本：5.4.3
+
+## 兼容性变更
+
```
Comment (TomShawn) [inline releases/release-5.4.3.md:12]:
```suggestion
```

File: releases/release-5.4.3.md [hunk 13]
```diff
@@ -0,0 +1,145 @@
...
++ TiKV **owner: @tabokie, TW: @Oreoxmt**
+
+    - 修复 Web 身份提供程序 (web identity provider) 报错并失效后，自动恢复为默认提供程序 (default provider) 时出现权限拒绝的问题 [#13122](https://github.com/tikv/tikv/issues/13122)
+    - (dup) 修复 PD 客户端可能会出现死锁的问题 [#13191](https://github.com/tikv/tikv/issues/13191)
+    - (dup) 修复 PD Region heartbeat 连接异常中断后未重新连接的问题 [#12934](https://github.com/tikv/tikv/issues/12934)
+    - (dup) 修复 Raftstore 线程繁忙时，可能会出现 Region 重叠的问题 [#13160](https://github.com/tikv/tikv/issues/13160)
+
++ PD **owner: @nolouch, TW: @Oreoxmt**
+
+    - 修复 PD 无法正确处理 dashboard 代理请求的问题 [#5321](https://github.com/tikv/pd/issues/5321)
```
Comment (Oreoxmt) [inline releases/release-5.4.3.md:99]: @nolouch PTAL

File: releases/release-5.4.3.md [hunk 14]
```diff
@@ -0,0 +1,145 @@
...
+    - (dup) 修复在某些情况下 `UPDATE` 语句错误地消除了 projection 导致 `Can't find column` 报错的问题 [#37568](https://github.com/pingcap/tidb/issues/37568)
+    - (dup) 修复某些情况下分区表无法充分利用索引来扫描数据的问题 [#33966](https://github.com/pingcap/tidb/issues/33966)
+    - 修复某些情况下，`EXECUTE` 语句可能抛出非预期异常的问题 [#37187](https://github.com/pingcap/tidb/issues/37187)
+    - 修复开启 Prepared Plan Cache 后，使用 `BIT` 类型的索引可能会导致查询结果错误的问题 [#33067](https://github.com/pingcap/tidb/issues/33067)
+
+    <!--diagnosis **owner: @mornyx**-->
+
++ TiKV **owner: @tabokie, TW: @Oreoxmt**
+
+    - 修复 Web 身份提供程序 (web identity provider) 报错并失效后，自动恢复为默认提供程序 (default provider) 时出现权限拒绝的问题 [#13122](https://github.com/tikv/tikv/issues/13122)
```
Comment (Oreoxmt) [inline releases/release-5.4.3.md:92]: @tabokie PTAL

File: releases/release-5.4.3.md [hunk 15]
```diff
@@ -0,0 +1,145 @@
...
+
+    <!--planner **owner: @winoros**-->
+
+    - (dup) 修复 Static Partition Prune 模式下带聚合条件的 SQL 语句在表为空时结果错误的问题 [#35295](https://github.com/pingcap/tidb/issues/35295)
+    - (dup) 修复 TiDB 在执行 `UPDATE` 语句时可能会 panic 的问题 [#32311](https://github.com/pingcap/tidb/issues/32311)
+    - (dup) 修复 `UnionScan` 无法保序导致的查询结果不正确的问题 [#33175](https://github.com/pingcap/tidb/issues/33175)
+    - (dup) 修复在某些情况下 `UPDATE` 语句错误地消除了 projection 导致 `Can't find column` 报错的问题 [#37568](https://github.com/pingcap/tidb/issues/37568)
+    - (dup) 修复某些情况下分区表无法充分利用索引来扫描数据的问题 [#33966](https://github.com/pingcap/tidb/issues/33966)
+    - 修复某些情况下，`EXECUTE` 语句可能抛出非预期异常的问题 [#37187](https://github.com/pingcap/tidb/issues/37187)
+    - 修复开启 Prepared Plan Cache 后，使用 `BIT` 类型的索引可能会导致查询结果错误的问题 [#33067](https://github.com/pingcap/tidb/issues/33067)
```
Comment (Oreoxmt) [inline releases/release-5.4.3.md:86]: @winoros PTAL

File: releases/release-5.4.3.md [hunk 16]
```diff
@@ -0,0 +1,145 @@
...
+    <!--execution **owner: @zanmato1984**-->
+
+    - (dup) 修复 `UNION` 运算符可能会非预期地返回空结果的问题 [#36903](https://github.com/pingcap/tidb/issues/36903)
+    - (dup) 修复在 TiFlash 中为分区表开启动态模式时结果出错的问题 [#37254](https://github.com/pingcap/tidb/issues/37254)
+    - (dup) 修复 `INL_HASH_JOIN` 和 `LIMIT` 一起使用时可能会卡住的问题 [#35638](https://github.com/pingcap/tidb/issues/35638)
+    - (dup) 修复执行 `SHOW WARNINGS` 时可能会报 `invalid memory address or nil pointer dereference` 的问题 [#31569](https://github.com/pingcap/tidb/issues/31569)
+
+    <!--transaction **owner: @cfzjywxk**-->
+
+    - 修复在读已提交隔离级别 (RC isolation level) 执行 Stale Read 报 `invalid transaction` 错误的问题 [#30872](https://github.com/pingcap/tidb/issues/30872)
```
Comment (Oreoxmt) [inline releases/release-5.4.3.md:74]: @cfzjywxk PTAL

File: releases/release-5.4.3.md [hunk 17]
```diff
@@ -0,0 +1,145 @@
...
+
+    <!--transaction **owner: @cfzjywxk**-->
+
+    <!--planner **owner: @winoros**-->
+
+    <!--diagnosis **owner: @mornyx**-->
+
++ TiKV **owner: @tabokie, TW: @Oreoxmt**
+
+    - 支持将 RocksDB write stall 设置为比流量控制阈值更小的值 [#13467](https://github.com/tikv/tikv/issues/13467)
```
Comment (Oreoxmt) [inline releases/release-5.4.3.md:29]: @tabokie PTAL

**Review by winoros — APPROVED** [review]

File: releases/release-5.4.3.md [hunk 18]
```diff
@@ -0,0 +1,100 @@
...
+
+发版日期：2022 年 x 月 x 日
+
+TiDB 版本：5.4.3
+
+## 提升改进
+
++ TiKV
+
+    - 支持将 RocksDB write stall 设置为比流量控制阈值更小的值 [#13467](https://github.com/tikv/tikv/issues/13467)
```
Comment (tabokie) [inline releases/release-5.4.3.md:15]:
```suggestion
    - 支持将 RocksDB write stall 参数设置为比 flow control 流控阈值更小的值 [#13467](https://github.com/tikv/tikv/issues/13467)
```

**Review by nolouch — APPROVED** [review]

**Review by tabokie — APPROVED** [review]

File: releases/release-5.4.3.md [hunk 19]
```diff
@@ -0,0 +1,100 @@
...
+    - 修复在某些情况下 `UPDATE` 语句错误地消除了 projection 导致 `Can't find column` 报错的问题 [#37568](https://github.com/pingcap/tidb/issues/37568)
+    - 修复某些情况下分区表无法充分利用索引来扫描数据的问题 [#33966](https://github.com/pingcap/tidb/issues/33966)
+    - 修复某些情况下，`EXECUTE` 语句可能抛出非预期异常的问题 [#37187](https://github.com/pingcap/tidb/issues/37187)
+    - 修复开启 Prepared Plan Cache 后，使用 `BIT` 类型的索引可能会导致查询结果错误的问题 [#33067](https://github.com/pingcap/tidb/issues/33067)
+
++ TiKV
+
+    - 修复 Web 身份提供程序 (web identity provider) 报错并失效后，自动恢复为默认提供程序 (default provider) 时出现权限拒绝的问题 [#13122](https://github.com/tikv/tikv/issues/13122)
+    - 修复 PD 客户端可能会出现死锁的问题 [#13191](https://github.com/tikv/tikv/issues/13191)
+    - 修复 PD Region heartbeat 连接异常中断后未重新连接的问题 [#12934](https://github.com/tikv/tikv/issues/12934)
```
Comment (Oreoxmt) [inline releases/release-5.4.3.md:56]:
```suggestion
    - 修复了 PD leader 发生切换或重启 PD 后，在集群中执行 SQL 语句会出现持续报错的问题 [#12934](https://github.com/tikv/tikv/issues/12934)
        - 问题原因：该问题是由于 TiKV 存在 bug，TiKV 向 PD client 发送心跳请求失败后不会重试，只能等待与 PD client 重连。这样，故障 TiKV 节点上的 Region 的信息会逐步变旧，使得 TiDB 无法获取最新的 Region 信息，导致 SQL 执行出错。
        - 影响版本：v5.3.2 和 v5.4.2。目前该问题已在 v5.3.3 和 v5.3.4 上修复。如果你使用 v5.3.2 的 TiDB 集群，可以升级至 v5.3.3 或以上版本。
        - 规避方法：除升级外，你还可以重启无法向 PD 发送 Region 心跳的 TiKV 节点，直至不再有待发送的 Region 心跳为止。
```

Comment (TomShawn) [reply releases/release-5.4.3.md:56]:
```suggestion
    - 修复了 PD leader 发生切换或重启 PD 后，在集群中执行 SQL 语句会出现持续报错的问题 [#12934](https://github.com/tikv/tikv/issues/12934)
	        - 问题原因：该问题是由于 TiKV 存在 bug，TiKV 向 PD client 发送心跳请求失败后不会重试，只能等待与 PD client 重连。这样，故障 TiKV 节点上的 Region 的信息会逐步变旧，使得 TiDB 无法获取最新的 Region 信息，导致 SQL 执行出错。
	        - 影响版本：v5.3.2 和 v5.4.2。目前该问题已在 v5.3.3 和 v5.3.4 上修复。如果你使用 v5.4.2 的 TiDB 集群，可以升级至 v5.4.3。
	        - 规避方法：除升级外，你还可以重启无法向 PD 发送 Region 心跳的 TiKV 节点，直至不再有待发送的 Region 心跳为止。
```

**Pull request merged**